h2o-bindings/bin/custom/R/gen_stackedensemble.py

```diff
@@ -38,6 +37,8 @@ def update_param(name, param):
 for (i in 1:length(base_models)) {
   if (inherits(base_models[[i]], 'H2OModel')) {
     base_models[[i]] <- base_models[[i]]@model_id
+  } else if (inherits(base_models[[i]], 'H2OGrid')) {
+    base_models[[i]] <- base_models[[i]]@grid_id
```

Comment:
can you please use the generic `h2o.keyof` instead? I created it for similar purposes, but apparently forgot to add it for grid.
cf `classes.R`:
```
setMethod("h2o.keyof", signature("H2OModel"), function(object) object@model_id)
```
you can add the equivalent for "H2OGrid" lower in the file, and simplify this:
```
if (inherits(base_models[[i]], c("H2OModel", "H2OGrid") 
   base_models[[i]] <- h2o.keyof(base_models[[i]])
```

Reply:
Good to know about this method. 

I have question about R coding style. I can see that you don't use curly brackets here for the single line expression. Is that to prevent people subconsciously adding some more lines of code in the if expr.? Or is it just to keep the example shorter?

Reply:
> to keep the example shorter

personally I'm using brackets for `block` statements to distinguish clearly from expressions (same for inline functions).
Looks like it is compatible with tidyverse: https://style.tidyverse.org/syntax.html#indenting